lib/elixir/lib/file.ex

```diff
@@ -120,15 +120,21 @@ defmodule File do
   This function follows symbolic links, so if a symbolic link points to a
   regular file, `true` is returned.
 
+  ## Options
+
+   - The list of options currently supports a single option, the atom `:raw`,
```

Comment:
I think the format @ericmj was suggesting was similar to [this one](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/file.ex#L382-L387). It's a common style in the Elixir codebase :)